corehq/apps/locations/tests/test_api_v6.py

```diff
@@ -0,0 +1,111 @@
+import json
+from corehq.apps.api.tests.utils import APIResourceTest
+from corehq.apps.locations.models import LocationType, SQLLocation
+from corehq.apps.locations.resources.v0_6 import LocationResource
+from corehq.util.view_utils import absolute_reverse
+
+
+class LocationV6Test(APIResourceTest):
+    api_name = 'v0.6'
+    resource = LocationResource
+
+    def setUp(self):
+        self.parent_type = LocationType.objects.create(
+            domain=self.domain.name,
+            name="State",
+            code="state",
+        )
+
+        self.child_type = LocationType.objects.create(
+            domain=self.domain.name,
+            name="City",
+            code="city",
+            parent_type=self.parent_type
+        )
+
+        self.location1 = SQLLocation.objects.create(
+            domain=self.domain.name,
+            id="1234",
+            location_id="1",
+            name="Colorado",
+            site_code="colorado",
+            location_type=self.parent_type
+        )
+
+        self.location2 = SQLLocation.objects.create(
+            domain=self.domain.name,
+            id="1235",
+            location_id="2",
+            name="Denver",
+            site_code="denver",
+            longitude=10.1234567891,
+            latitude=11.1234567891,
+            location_type=self.child_type,
+            parent=self.location1,
+            metadata={"population": "715,522"}
+        )
+
+    def single_endpoint(self, id):
```

Comment:
I'd use `location_id` instead of `id` since the latter usually means the Django primary key.

(aside: I really wish we just used the UUID as the primary key, but I'm not annoyed enough to try to change it)